rdt-open-reader/src/covidIndonesia/assets/json.form/patient-registration-form.json

```diff
@@ -521,10 +521,23 @@
         "openmrs_entity": "",
         "openmrs_entity_id": "",
         "type": "edit_text",
+        "edit_type": "number",
         "top_padding": "0dp",
         "v_required": {
           "value": "false",
           "err": "{{patient-registration-form-original.step1.patient_whatsapp.v_required.err}}"
+        },
+        "v_numeric": {
```

Comment:
@yussin-hidayat was this generated using JMAG? The strings are not interpolated in the format `{{string}}`

Reply:
Oops.. fixed.